jobs/scrape_archives.py

```diff
@@ -178,33 +185,33 @@ def archive_url(product, version=None, platform=None, locale=None, nightly=None)
 
     # Skip aurora nightlies and known nightlies...
     days_urls = [archive_url(product, nightly=current_month + "/" + f[:-1])
-                 for f in days_folders if f > latest_nightly_folder and f.endswith("mozilla-central/")]
+                 for f in days_folders if f > latest_nightly_folder]
+    days_urls = [url for url in days_urls if "mozilla-central" in url]
+
     futures = [fetch_listing(session, day_url) for day_url in days_urls]
     listings = await asyncio.gather(*futures)
 
     channel = "nightly"
+    re_filename = re.compile(r"\w+-(\d+.+)\.([a-z]+(\-[A-Z]+)?)\.(.+)\.({})$".format(FILE_EXTENSIONS))
 
     for day_url, (_, files) in zip(days_urls, listings):
-        futures = []
         for file_ in files:
             filename = file_["name"]
             size = file_["size"]
             date = file_["last_modified"]
             url = day_url + filename
 
-            match = re.search(r'\w+-(\d+.+)\.([a-z]+(\-[A-Z]+)?)\.(.+)\.tar.bz2+$', filename)
-            if not match:
+            match = re_filename.search(filename)
+            if not match or "tests" in filename:
                 continue
             version = match.group(1)
             locale = match.group(2)
             platform = match.group(4)
 
             metadata = await fetch_nightly_metadata(session, url)
             record = archive(product, version, platform, locale, channel, url, size, date, metadata)
-            logger.debug("Nightly found %s" % url)
-
-            futures.append(queue.put(record))
-        await asyncio.gather(*futures)
+            logger.debug("Nightly found {}".format(url))
+            await queue.put(record)
```

Comment:
What's this change about? Doesn't this make the queue operation more synchronous?